dev-cpp/exprtk/exprtk-0.0.3.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit edo toolchain-funcs
+
+DESCRIPTION="C++ Mathematical Expression Parsing And Evaluation Library"
+HOMEPAGE="
+	https://www.partow.net/programming/exprtk/index.html
+	https://github.com/ArashPartow/exprtk/
+"
+SRC_URI="
+	https://github.com/ArashPartow/exprtk/archive/refs/tags/${PV}.tar.gz
+		-> ${P}.tar.gz
+"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+
+DOCS=( readme.txt )
+
+src_compile() { :; }
+
+src_test() {
+	emake \
+		COMPILER="$(tc-getCXX)" \
+		OPTIMIZATION_OPT="${CXXFLAGS}" \
+		LINKER_OPT="-L/usr/$(get_libdir) -lm ${LDFLAGS}" \
```

Comment:
You shouldn't ever need `-L/usr/$(get_libdir)` because it should be in the toolchain's default search path. But absolute `/usr` would be wrong on Prefix anyway.